### PR TITLE
Week 5 Quiz submission from asharda-stat6250

### PR DESCRIPTION
**Pull request by asharda-stat6863**
Week 5 Quiz submission from asharda-stat6250

### CODE REVIEW

File: week-05/README.md [hunk 1]
```diff
@@ -32,6 +32,12 @@ The instructor will then review the pull request and make comments should furthe
 - Question(dfei-stat6250): What's the rule of SAS statement associates the fileref Crime with the raw data file?
 - Answer(dfei-stat6250): It is filename crime 'c:\states\data\crime.dat'
 - *Question* (aamiri2-stat6250): What are the steps in creating a raw data file? How can one identify a raw data file in a SAS statement?
+- *Answer* (asharda-stat6250):
```
Comment (prof-lankham) [inline week-05/README.md:35]: No bullet points or line breaks should be used in answers

File: week-05/README.md [hunk 2]
```diff
@@ -58,6 +64,8 @@ The instructor will then review the pull request and make comments should furthe
 
 [Course Textbook Chapter 5, Problem 2]
 - *Question* (cli19-stat6250): What is the best practice for maintaining filerefs when programming?
+- *Answer* (asharda-stat6250): FILENAME statements are global; they remain in effect until you change them, cancel them,
+or end your SAS session.
```
Comment (prof-lankham) [inline week-05/README.md:68]: No bullet points or line breaks should be used in answers

**Review by prof-lankham — CHANGES_REQUESTED** [review]
Please update PR and DM me in Slack with PR URL when ready for re-evaluation

**Comment by asharda-stat6863** [comment]
Did submit a new PR

**Pull request closed without merging**